src/logic/documents/scorecards.js

```diff
@@ -360,6 +368,37 @@ const scorecard = ({
   } = scorecardPaperSizeInfos[scorecardPaperSize];
   const scorecardWidth = pageWidth / scorecardsPerRow - 2 * horizontalMargin;
 
+  let printSecondScramblerBox = false;
+  if (
+    printSecondScramblerForTopRankedCompetitors ||
+    printSecondScramblerForFinalRounds
+  ) {
+    if (printSecondScramblerForTopRankedCompetitors) {
+      const singlePersonalBest = competitor.personalBests?.find(
+        personalBest =>
+          personalBest.eventId === eventId && personalBest.type === 'single'
+      );
+      const averagePersonalBest = competitor.personalBests?.find(
+        personalBest =>
+          personalBest.eventId === eventId && personalBest.type === 'average'
+      );
+      if (
+        singlePersonalBest?.worldRanking <= 100 ||
+        singlePersonalBest?.nationalRanking <= 25 ||
```

Comment:
If `singlePersonalBest` is undefined, this becomes `undefined <= 100`. It evaluates to `false`, but I would prefer to not rely on this behaviour implicitly. I would suggest to make this more explicit as in:

```
singlePersonalBest && (singlePersonalBest.worldRanking <= 100 || singlePersonalBest.nationalRanking <= 25)
```